sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -88,7 +93,7 @@ internal DevicesClient(RegistryManagerRestClient registryManagerClient, TwinRest
         /// <returns>The retrieved device identity and the http response <see cref="Response{T}"/>.</returns>
         public virtual Task<Response<DeviceIdentity>> GetIdentityAsync(string deviceId, CancellationToken cancellationToken = default)
         {
-            return _registryManagerClient.GetDeviceAsync(deviceId, cancellationToken);
+            return _devicesRestClient.GetDeviceAsync(deviceId, cancellationToken);
```

Comment:
Everything above is updated to end with Identity but not GET. I will be requesting for that change.